src/pages/EmployeeSkillsPage/EmployeeSkillsPage.tsx

```diff
@@ -1,7 +1,121 @@
+import { useMutation } from '@apollo/client';
+import { Box, Chip, Typography } from '@mui/material';
+import { useForm } from 'react-hook-form';
+
 import EmployeeTabs from '../../components/EmployeeTabs';
+import Preloader from '../../components/Preloader';
+import { UPDATE_USER } from '../../graphql/user/mutation';
+import { USER } from '../../graphql/user/query';
+import useUserData from '../../hooks/useUserData';
+import isAbleToEdit from '../../utils/isAbleToEdit';
+import {
+  StyledBox,
+  StyledButton,
+  StyledForm,
+  StyledFormControl,
+  StyledInputLabel,
+  StyledOutlinedInput
+} from './EmployeeSkillsPage.styles';
+
+interface IFormInput {
+  skillName: string;
+  mastery: string;
+}
 
 const EmployeeSkillsPage: React.FC = () => {
-  return <EmployeeTabs />;
+  const { user, loggedUser, loading, error } = useUserData();
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    reset
+  } = useForm<IFormInput>({
+    defaultValues: {
+      skillName: '',
+      mastery: ''
+    }
+  });
+
+  const [updateUserMutation, { loading: updateUserLoading }] = useMutation(
+    UPDATE_USER,
+    {
+      refetchQueries: [{ query: USER, variables: { id: user?.id } }]
+    }
+  );
+
+  const ableToEdit = isAbleToEdit(loggedUser, user);
+
+  const onSubmit = async ({ skillName, mastery }: IFormInput) => {
+    reset();
+
+    // doesn't work because of the backend
+    await updateUserMutation({
+      variables: {
+        id: user?.id,
+        user: {
+          profile: {
+            skills: [{ skill_name: skillName, mastery }]
+          }
+        }
+      }
+    });
+  };
+
+  return (
+    <Preloader loading={loading || updateUserLoading} error={error}>
+      <>
+        <EmployeeTabs />
+        <Box width={750} margin="auto" marginY={6}>
```

Comment:
I see this line several times. MAybe let's create styled component. Will be easye to mainntain it